src/driver/mysql/MysqlDriver.ts

```diff
@@ -509,13 +509,17 @@ export class MysqlDriver implements Driver {
         if (column.length)
             return column.length.toString();
 
+        /**
+         * fix https://github.com/typeorm/typeorm/issues/1139
+         */
+        if (column.generationStrategy === 'uuid')
```

Comment:
could you please provide a test for this change? Also, it would be nice if you could add a record in changelog and place this PR against `next` branch